File: src/main/java/FlyAway/flight/dto/FlightSummaryEmployeeDto.java
package FlyAway.flight.dto;

import FlyAway.flight.aircraft.CabinClass;
import FlyAway.flight.airport.dto.AirportCityIataCodeDto;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.Map;
import java.util.UUID;

public record FlightSummaryEmployeeDto(
        UUID id,
        AirportCityIataCodeDto departureAirportDto,
        AirportCityIataCodeDto arrivalAirportDto,
        LocalDateTime departureDate,
        LocalDateTime arrivalDate,
        Map<CabinClass, BigDecimal> cabinClassPrices
) {
}

File: frontend/src/app/services/models/flight-summary-employee-dto.ts
/* tslint:disable */
/* eslint-disable */
import { AirportCityIataCodeDto } from '../models/airport-city-iata-code-dto';
export interface FlightSummaryEmployeeDto {
  arrivalAirportDto?: AirportCityIataCodeDto;
  arrivalDate?: string;
  cabinClassPrices?: {
[key: string]: number;
};
  departureAirportDto?: AirportCityIataCodeDto;
  departureDate?: string;
  id?: string;
}
